app/client/cli/debug.go

```diff
@@ -165,12 +162,13 @@ func sendDebugMessage(debugMsg *messaging.DebugMessage) {
 	}
 
 	var validatorAddress []byte
-	for _, val := range consensusMod.ValidatorMap() {
-		validatorAddress, err = pocketCrypto.NewAddress(val.GetAddress())
-		if err != nil {
-			log.Fatalf("[ERROR] Failed to convert validator address into pocketCrypto.Address: %v", err)
-		}
-		break
+	if len(validators) == 0 {
```

Comment:
🤔 I disagree, at some point, hopefully VERY soon, we'll have dynamic node set and we could (also for testing purposes) just don't have any addressable node to connect to when we send.

I would keep the guard here so that whoever touches this code is aware of that eventuality at runtime.
I wouldn't  even bother adding a check in `initDebug` because for now it's straight from genesis and it doesn't change.